tests/docker/ducktape-deps.sh

```diff
@@ -37,6 +37,7 @@ function install_system_deps() {
     npm \
     openssh-server \
     netcat-openbsd \
+    protobuf-compiler \
```

Comment:
I wonder if we can/should use the one we build in the build, e.g.:
```
vbuild/debug/clang/rp_deps_install/bin/protoc
```

Reply:
The docker container is built independent of Redpanda.  Your local build of Redpanda is mounted within the containers when the cluster is composed.

Reply:
I've had huge problems getting an arbitrary version of `protoc` to generate code that works with an arbitrary version of a protobuf library dependency. That's across C++, Go, and Python.  Maybe I'm just doing something wrong.

Reply:
I'm not as familiar with protobufs as I am with flatbuffers... I'm compiling with version A has such issues working with library version B, if they are at least "close".

Maybe the correct solution for docker, and anything, is to build and use the same protoc across all use cases.  I can update the docker build script to do just that.

Reply:
Right, but protoc is already built. My well-known PR updates the Python lib to the same version as what we build for Redpanda. (Don't let the version number fool you).

Reply:
> Right, but protoc is already built.

The build for the test Docker image doesn't reference any build artifacts for Redpanda.  Instead, the container is mounted with your build directory as a volume after it is built.


> (Don't let the version number fool you).

It did